libs/components/forms/testing/src/input-box/input-box-harness.ts

```diff
@@ -166,21 +174,19 @@ export class SkyInputBoxHarness extends SkyQueryableComponentHarness {
   }
 
   /**
-   * Gets the help popover for the input box or throws an error if
-   * the help popover is not configured.
+   * Gets the help popover content.
    */
-  public async getHelpPopover(): Promise<SkyPopoverHarness> {
```

Comment:
We've already documented this method in [our public docs](https://developer.blackbaud.com/skyux/components/input-box#class-skyinputboxharness), so removing this is a breaking change. What's the reasoning for removing it? If we don't remove it, can we rely on it for consumers to click the help inline button rather than having a first-class method on the harness to do so?

Reply:
my intent behind the change was to make the input harness functions to interact with help inline popover match what we do in other components. if we do keep this functionality, we would change the harness we return to the new `SkyHelpInlineHarness` on the new component which isn't public yet. i also remember with the form error work that instead of exposing the underlying harness, we made first class functions for each component that used the form error harness under the hood.

Reply:
from standup: we dont want to remove functionality, so keep this function (change working under the hood to expose the SkyPopoverHarness). But add the new functions to bring input box to the same implementation as other components